pkg/minishift/registration/utils.go

```diff
@@ -47,7 +40,7 @@ func doRegistrationHostAction(actionMessage string, registrationAction Registrat
 		return false, err
 	}
 
-	// was succesfully
+	// was successfully
```

Comment:
if you change it, please correct it to `was successful` (no adjective form, as I actually forgot to add `completed`)